osu.Game.Rulesets.Taiko/Audio/DrumSampleMapping.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using osu.Framework.Audio;
+using osu.Framework.Audio.Sample;
+using osu.Game.Audio;
+using osu.Game.Beatmaps.ControlPoints;
+
+namespace osu.Game.Rulesets.Taiko.Audio
+{
+    public class DrumSampleMapping : IComparable<DrumSampleMapping>
+    {
+        public double Time;
+        public readonly SampleInfo Centre;
+        public readonly SampleInfo Rim;
+
+        public SampleChannel CentreChannel { get; private set; }

```

Comment:
as discussed today, this is not what we want. it will result in the previous playback being interrupted by new ones

Reply:
As discussed, SampleChannelBass creates a new channel for every time Play() is invoked on the SampleChannelBass, so this won't lead to clipping.

Reply:
yep, confusing class names